### PR TITLE
Integrated final

### PR DESCRIPTION
**Pull request by shahp98**
Integrated the login and registration page successfully.

### CODE REVIEW

**Comment by AvaisP** [comment]
@shahp98 Fix the merge conflicts

**Comment by AvaisP** [comment]
@viral98 Modify gitignore to ignore all .idea files and all bundles. We can't have 14000 files in the PR

**Comment by shahp98** [comment]
added **/.idea/* to .gitignore
i dont know about the bundles....
solving package-lock.json ka conflicts right now... its a pain
conflicts solve karke .idea ka wapis se commit daalta hun

**Comment by ikbalsingh** [comment]
Add npm scripts to package.json for building and running the thing.

**Comment by shahp98** [comment]
they're already there in dependencies and devDependencies of package.json
where do you want us to add them?

**Comment by ikbalsingh** [comment]
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "./node_modules/.bin/webpack .........(command)",
  }
aise add these scripts so that one just needs to do "npm run start".
Add one for build as well.

**Comment by ikbalsingh** [comment]
image
Is the css for the form not loading? Also there was a unicode ka logo right?

**Comment by AvaisP** [comment]
Why is the build failing? Please fix it

**Comment by AvaisP** [comment]
Why have we shifted so many files from src to assets?

**Comment by shahp98** [comment]
@AvaisP, have theek che? :P 
i still have the scripts wala kaam pending... woh karke final thing push karta hun

**Comment by AvaisP** [comment]
@shahp98 @viral98 Setup instructions please

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ikbalsingh) [inline package.json]: Add a comma (,) at the end.

File: assets/bundles/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.js
```
Comment (AvaisP) [inline assets/bundles/.gitignore]: Add new line

**Comment by AvaisP** [comment]
Add *.pyc to gitignore

**Comment by AvaisP** [comment]
What is secrets.sh?

**Comment by AvaisP** [comment]
Rename all images from hash.png to filename.png

**Comment by AvaisP** [comment]
Why do we have same images in users/static and assets/bundles? Instead add collectstatic in setup instructions

**Comment by ikbalsingh** [comment]
image
Once clicked register, cannot go back to login form. I can login but I will  have to fill name and rest of the registration fields. Why don't you render the login form again or remove those registration fields.

**Comment by shahp98** [comment]
basically you want me to remove pycache right?

**Comment by shahp98** [comment]
https://github.com/sf-wdi-31/how-to-hide-secrets

for secrets.sh
basically it is used to store the key and keep it a secret

**Comment by shahp98** [comment]
I'm not sure if the following points in the readme should be replaced.
6) [Line 10] create a .env according to .env.sample
10) [Line 14] npm start

**Comment by ikbalsingh** [comment]
The register button remains disabled inspite of no errors.

**Comment by ikbalsingh** [comment]
Also the login button is not sending a POST request but is sending a GET and so Login is also not working.

**Comment by ikbalsingh** [comment]
Now the login button remains disabled inspite of no errors.

**Comment by ikbalsingh** [comment]
image
This is what happens after register is clicked.

**Pull request closed without merging**